wiki/Beatmap_Editor/AiMod/en.md

```diff
@@ -25,91 +25,91 @@ Development of AiMod usually lags behind the standards held by the mapping commu
 
 #### Meta
 
-| Message | Explantation | Solution |
+| Message | Explanation | Solution |
 | :-- | :-- | :-- |
-| HP rate for Easy/Normal is suggested to be at least 4. | osu!mania maps only! | Go to Song Settings and set HP rate to 4 or higher |
-| HP rate for Hard and up is suggested to be at least 7. | osu!mania maps only! | Go to Song Settings and set HP rate to 7 or higher |
-| OD rate for maps with few sliders is suggested to be at least 7. | osu!mania maps only! | Go to Song Settings and set OD rate to 7 or higher |
-| The Slider Velocity should be 1.40 or 1.60. | osu!taiko maps only! | Go to Timing tab and change Slider Velocity to 1.40 or 1.60 |
+| HP rate for Easy/Normal is suggested to be at least 4. | osu!mania maps only! | Go to Song Settings and set HP rate to 4 or higher. |
+| HP rate for Hard and up is suggested to be at least 7. | osu!mania maps only! | Go to Song Settings and set HP rate to 7 or higher. |
+| OD rate for maps with few sliders is suggested to be at least 7. | osu!mania maps only! | Go to Song Settings and set OD rate to 7 or higher. |
+| The Slider Velocity should be 1.40 or 1.60. | osu!taiko maps only! | Go to Timing tab and change Slider Velocity to 1.40 or 1.60. |
 
 ### Errors
 
 #### Compose
 
-| Message | Explantation | Solution |
+| Message | Explanation | Solution |
 | :-- | :-- | :-- |
 | These two objects are less than 10ms apart! | Two objects are very close to each other. This means that the player will need to click at an impossible rate! | Find the two hit objects and move it or delete it. |
 | These two objects exist at the same point in time! | Two objects are on top of each other. This will make the map impossible to rank! | Find the two hit objects and move it or delete it. |
-| There are no hitsounds on any objects. Hitsounds are required for rankable beatmaps. |  | Hitsound the map using whistles, claps and finishes. |
+| There are no hitsounds on any objects. Hitsounds are required for rankable beatmaps. | (Self-explanatory) | Hitsound the map using whistles, claps and finishes. |
 | This spinner appears onscreen later than objects which follow it. | Objects appear while the spinner is active. This could be caused by having low AR and/or placing objects too close (time-wise) to the spinner. | Change the length of the spinner and/or remove the following objects |
-| This hold note is less than 10ms long! | osu!mania maps only! This will require the player to press and release the key at a very quick rate, making your map impossible to obtain an SS. |  |
-| This object overlaps with another object. | osu!mania maps only! |  |
+| This hold note is less than 10ms long! | osu!mania maps only! This will require the player to press and release the key at a very quick rate, making your map impossible to obtain an SS. | Change the length of the hold note or remove it. |
```

Comment:
```suggestion
| This hold note is less than 10ms long! | osu!mania maps only! This will require the player to press and release the key at a very quick rate, making your map impossible to obtain an SS. | Increase length of the hold note or remove it. |
```